tests/configure.ac

```diff
@@ -269,6 +269,12 @@ AC_ARG_ENABLE([easybuild],
    [AX_OPTION_DEFAULT(type=long,enable_easybuild,yes)])
 AM_CONDITIONAL(EASYBUILD_ENABLED,test "x$enable_easybuild" = "xyes" ) 
 #------------------------------------------------------------------------------------------
+# cuda
+AC_ARG_ENABLE([cuda],
+   [AS_HELP_STRING([--enable-cuda],[Enable cuda tests (default=yes)])],[],
+   [AX_OPTION_DEFAULT(type=long,enable_cuda,no)])
```

Comment:
The line above says `default=yes` but here it says `no`

Reply:
Thanks, will fix it.